package.json

```diff
@@ -56,15 +56,14 @@
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
     "react-final-form": "^6.3.0",
-    "react-intl": "^2.8.0",
+    "react-intl": "^5.8.2",
```

Comment:
this can be updated only when we update  terra-application-navigation and @cerner/terra-core-docs to remove terra-base and terra-i18n as mentioned on this PR : https://github.com/cerner/terra-ui/pull/513

Reply:
@supreethmr thanks for advice, but this is still blocked by issue I described in comments here: https://jira2.cerner.com/browse/UXPLATFORM-6973

does your suggestion helps for solving?